tests/linked_list_test.py

```diff
@@ -28,10 +32,10 @@ def test_search_returns_false_for_empty_list(list):
     assert list.search(3) == False
 
 
-def test_search_returns_false_for_items_not_in_list(list):
-    list.add_first(3)
-    list.add_first(4)
-    assert list.search(5) == False
+# def test_search_returns_false_for_items_not_in_list(list):
+#     list.add_first(3)
+#     list.add_first(4)
+#     assert list.search(5) == False
```

Comment:
This is why it fails in learn.  You commented this out in the original tests.

Your function returns `None` for items not in the list rather than `False`.